tests/unit/test_hyper_transformer.py

```diff
@@ -280,13 +275,12 @@ def test__learn_config(self, get_default_transformer_mock):
             'categorical': 'categorical',
             'datetime': 'datetime'
         }
-        assert ht.field_transformers == {
-            'integer': int_transformer,
-            'float': float_transformer,
-            'categorical': categorical_transformer,
-            'bool': bool_transformer,
-            'datetime': datetime_transformer
-        }
+
+        assert isinstance(ht.field_transformers['integer'], FloatFormatter)
+        assert isinstance(ht.field_transformers['float'], ClusterBasedNormalizer)
+        assert isinstance(ht.field_transformers['categorical'], FrequencyEncoder)
+        assert isinstance(ht.field_transformers['bool'], BinaryEncoder)
+        assert isinstance(ht.field_transformers['datetime'], UnixTimestampEncoder)
```

Comment:
Just minor but I think that here we could ensure that `ht.field_transformers['integer'] == instance_integer`.
I would probably define the transformers before setting them to the field transformers and here ensure that they match the same id rather than just being an instance of those.

Reply:
`learn_config` uses o copy of the transformers, not the transformers themselves.